diesel/src/pg/query_builder/tablesample.rs

```diff
@@ -0,0 +1,170 @@
+use crate::{
+    expression::{Expression, ValidGrouping},
+    pg::Pg,
+    query_builder::{AsQuery, AstPass, FromClause, QueryFragment, QueryId, SelectStatement},
+    query_source::QuerySource,
+    result::QueryResult,
+    sql_types::{Double, SmallInt},
+    JoinTo, SelectableExpression, Table,
+};
```

Comment:
Minor style nit: Diesel uses unnested imports everywhere else, it would be great to keep that consistent.